bc_obps/reporting/api/report_additional_data.py

```diff
@@ -52,8 +50,9 @@ def save_report_additional_data(
 @handle_http_errors()
 def get_report_additional_data_by_version_id(
     request: HttpRequest, report_version_id: int
-) -> Tuple[Literal[200], ReportAdditionalDataOut]:
-    report_person_responsible = ReportAdditionalDataService.get_report_person_responsible_by_version_id(
+) -> tuple[Literal[200], ReportAdditionalData]:
```

Comment:
This needs to be a django ninja schema. Was there something missing from `ReportAdditionalDataOut` ?